ManPhamQuang/COSC2769-rmit
# refactored API calls

### Changelog
- Added a new custom axios instance for global settings.
- Added a new env file `next.config.js` (needs to change the baseApiUrl for production)

### How to test
- Checkout PR
- Start the backend server
- Run `npm run build` and verify that next js can build successfully
- Start a nextjs production server with `npm start`
- Navigate to `localhost:3000` and test out pre-existing features.
